smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SymbolVisitor.kt

```diff
@@ -97,7 +97,7 @@ class SymbolVisitor(private val model: Model, swiftSettings: SwiftSettings) :
     override fun toMemberName(shape: MemberShape): String {
         val containingShape = model.expectShape(shape.container)
         if (containingShape is UnionShape) {
-            val name = escaper.escapeMemberName(shape.memberName)
+            val name = escaper.escapeMemberName(shape.memberName.toLowerCamelCase())
```

Comment:
This was preventing the union member named `Enum` from being properly escaped in Swift code when it is downcased to the Swift keyword `enum`.